localstack/services/apigateway/provider.py

```diff
@@ -667,6 +667,14 @@ def import_rest_api(
             ),
         )
 
+    def delete_integration(
+        self, context: RequestContext, rest_api_id: String, resource_id: String, http_method: String
+    ) -> None:
+        try:
+            call_moto(context)
+        except Exception as e:
+            raise NotFoundException("Invalid Resource identifier specified") from e
```

Comment:
It would make sense to limit the exception clause a bit. You can inspect botocore exceptions by catching `CommonServiceException` and then checking `e.code` for the particular error code.

Reply:
Do you have an example of that exception handle practice?

Reply:
in most cases you can simply pass through the exception and don't need to do anything. if moto raises a `NotFoundException`, then `call_moto` will re-raise something that is equivalent and will be rendered correctly.

here's an example where we inspect the exception and re-raise them conditionally. not sure that's your use case though https://github.com/localstack/localstack/blob/5ee3711fc777ea5726d04bedc499d4c385fdbc11/localstack/services/dynamodb/provider.py#L466-L480

Reply:
I guess it makes sense depending on the case, as we talked in private is not applicable in this case.